File: mahaam-api-java/src/main/java/mahaam/feat/task/Task.java
package mahaam.feat.task;

import java.time.Instant;
import java.util.UUID;

public class Task {
	public UUID id;
	public UUID planId;
	public String title;
	public boolean done;
	public int sortOrder;
	public Instant createdAt;
	public Instant updatedAt;
}

File: mahaam-api-go/app/models/task.go
package models

import (
	"time"

	"github.com/google/uuid"
)

type Task struct {
	ID        uuid.UUID  `db:"id"`
	PlanID    uuid.UUID  `db:"plan_id"`
	Title     string     `db:"title"`
	Done      bool       `db:"done"`
	SortOrder int        `db:"sort_order"`
	CreatedAt *time.Time `db:"created_at"`
	UpdatedAt *time.Time `db:"updated_at"`
}
